publications/joss/paper.md

```diff
@@ -0,0 +1,129 @@
+telescopic observations from the ground or from space - utilizes a
+wide range of methods that are common in observational
+astronomy. However, some aspects, including the planning of
+observations, as well as the analysis and interpretation of the
+results, require tailored techniques and models that are disparate
+from those used most other astronomical fields of research. Currently,
+there is no single open source software package available to support
+small-body planetary astronomers in their study of asteroids and
+comets in the same way in which ``Astropy`` [@astropy; @astropy2]
+supports the general astronomy community.
+
+``sbpy`` is a community effort to build a Python package for
+small-body planetary astronomy in the form of an ``Astropy` affiliated
+package. The goal is to collect and implement well-tested and
+well-documented code for the scientific study of asteroids and comets,
+including (but not limited to):
+
+* observation planning tools tailored to moving objects;
+* photometry models for resolved and unresolved observations;
+* wrappers and tools for astrometry and orbit fitting;
+* spectroscopy analysis tools and models for reflected solar light and
+  emission from gas;
+* cometary gas and dust coma simulation and analysis tools;
+* asteroid thermal models for flux estimation and size/albedo estimation;
+* image enhancement tools for comet comae and PSF subtraction tools;
+* lightcurve and shape analysis tools;
+* access tools for various databases for orbital and physical data, as
+  well as ephemerides services.
+
+``sbpy`` is available and being maintained as a github repository at
+[github.com/NASA-Planetary-Science/sbpy](https://github.com/NASA-Planetary-Science/sbpy);
+documentation is available on
+[sbpy.readthedocs.io](https://sbpy.readthedocs.io/en/latest/)
+
+All functionality provided as part of ``sbpy`` has been tested against
+published results in order to ensure its correctness. An internal
+reference tracking system enables users to query a list of appropriate
+references depending on the functionality that has been utilized. In
+order to improve the performance of computationally intensive
+functions, like thermal modeling of atmosphereless bodies,
+C-extensions are utilized.
+
+Designed as an ``Astropy`` affiliated package, ``sbpy`` utilizes a
+wide range of functionality of ``Astropy``, including its table,
+constants, units, and modeling submodules, and other affiliated
+packages like [@astroquery]. ``sbpy``'s API is deliberately emulating
+that of ``Astropy`` wherever possible in order to enable user-friendly
+and consistent access to its functionality.
+
+``sbpy`` is setup in a highly modular class-based fashion, roughly
+separated into the individual tasks listed above. The following
+schematic provides an overview of these modules and their
+interrelations, as indicated through colored symbols of different
+shapes:
+
+![``sbpy`` module structure.](structure.png)
```

Comment:
Looks great! I could not get the image with the compilation command https://github.com/openjournals/whedon/blob/25f9a1307a83b6b89080d6d934a3621f6a244035/lib/whedon/processor.rb#L101-L122 , but I suspect that something has changed in the building process and that command may not be up to date, so better to leave this issue to the editors.

Reply:
I agree!